Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -1,127 +1,244 @@
-.. include:: /Includes.rst.txt
-.. index::
-   ! Path
-   see: Directory structure; Path
-.. _directory-structure:
+..  include:: /Includes.rst.txt
+..  index::
+    ! Path
+    see: Directory structure; Path
+..  _directory-structure:
 
 ===================
 Directory structure
 ===================
 
-By default a TYPO3 installation consists of a structure of
-main directories within the web server document root. You will find
-this structure to be almost always like that. Depending on the installation
-variant you choose however, this may be slightly different. For instance,
-it is possible to have all PHP files except the entry points :file:`index.php`
-within the Composer-managed :file:`vendor/` directory, outside of the document
-root. This setup however did not fully settle yet, and is not documented
-here in detail. So, if you look at "casual" TYPO3 installations, you will
-almost always find the directory structure as outlined below.
+The structure below describes the directory structure in a typical
+Composer-base TYPO3 installation. For the structure in a legacy installation
```

Comment:
```suggestion
Composer-based TYPO3 installation. For the structure in a legacy installation
```

Also, that this page is for Composer-based installations should be added to the heading: "Directory structure (Composer-based installations)" or something to make it clear. Not everyone reads the first sentence when he/she/it looks for a path, but the heading is more prominent.